examples/dlb/example.rst

```diff
@@ -10,6 +10,23 @@ This sandbox provides an example about how to enable DLB connection balanace.
 Step 1: Install DLB Driver
 **************************
 
+Please run below command to check your CPU supports DLB:
```

Comment:
i think lets put this a the top as a `.. note::` - does it require any specific kernel version?

Reply:
It depends on the DLB driver requirement, 5.15+ is enough.